### PR TITLE
auth page fixes

### PR DESCRIPTION
**Pull request by MeenuKumari28**
review fixes for auth [page](https://ui-git-auth-page-fixes-fifthtry.vercel.app/scenarios/slides/signin-page/)

### CODE REVIEW

**Review by yashveer29 — CHANGES_REQUESTED** [review]
Ui Reviews:-
1. Increase the "fifthtry slides" logo size
2. "Do not have acc" is broken fix the font
3. Change the button colour to white.

The default language should be English.

**Review by Arpita-Jaiswal — APPROVED** [review]

**Review by amitu — APPROVED** [review]

**Comment by amitu** [comment]
Waiting for @yashveer29's approval.

**Review by yashveer29 — APPROVED** [review]

**Pull request merged**
